applications/nrf5340_audio/src/audio_module/amod_api.c

```diff
@@ -0,0 +1,1136 @@
+/*
+ * Copyright(c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "amod_api.h"
+
+#include <stdlib.h>
+#include <stdio.h>
+#include <ctype.h>
+#include <zephyr/kernel.h>
+#include <zephyr/shell/shell.h>
+#include <nrfx_clock.h>
+#include "data_fifo.h"
+#include "amod_api_private.h"
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(amod_api, 3);
```

Comment:
Use Kconfig

Reply:
Yes I was just setting this to give logs, it is actually 4 now!